package.json

```diff
@@ -220,6 +220,7 @@
     "gulp": "4.0.2",
     "highlight-words": "1.2.0",
     "history": "4.10.1",
+    "html-attributes": "1.1.0",
```

Comment:
is this a required import here? can't see it referenced/imported anywhere but yarn.lock

Reply:
removed!